### DISCUSSION TITLE
Unable to sign up with Google in Armo portal

### DISCUSSION THREAD

**Discussion by dwisiswant0** [kubescape/kubescape#204] [Q&A]
Hi, I'm trying to register an account on the Armo portal but I'm having trouble.

Initially, I have tried to log in using a Google account, but after a callback from `accounts.google.com` - the portal doesn't show dashboard/anything _(only splash loading)_. When I want to re-register the account with email (not with SSO), it says my account has been registered.

Can you guys assist me with this? Thanks in advance.

**Comment by slashben** [comment]
Hi @dwisiswant0 

I have tried the same, signing up with a new Google account and it worked for me. 

Do you mind sharing your e-mail address so I can see if we created the account in our side?

Thx
B
